tools/testdata-scripts/Cargo.toml

```diff
@@ -24,3 +24,10 @@ rust-format = { version = "0.3.4", features = ["token_stream"] }
 simple_logger = { version = "4.1.0", default-features = false }
 ureq = "2"
 zip = ">=0.5, <0.7"
+
+[features]
+make-testdata-legacy = ["icu_datagen/legacy_api"]
```

Comment:
Question: the old API is covered but is it being run anywhere in CI? Do we care?

Reply:
We care to not break semver. It's only checked, not executed.